plugins/app-backend/src/service/router.ts

```diff
@@ -95,9 +95,28 @@ export async function createRouter(
   staticRouter.use(notFoundHandler());
 
   router.use('/static', staticRouter);
-  router.use(express.static(appDistDir));
+  router.use(
+    express.static(appDistDir, {
+      setHeaders: (res, path) => {
+        // The Cache-Control header instructs the browser to not cache html files since it might
+        // link to static assets from recently deployed versions. This is a workaround when no
+        // staticFallbackHandler is configured.
+        // use `as any` since express uses mime v1 while we only have types for mime v2
```

Comment:
Doh 😅 , perhaps we could just create a trivial `{lookup(string): string}` type for it in this file? Just to avoid any. Even that kinda simple thing has helped me in the past :grin: